lib/card_search_query_builder.rb

```diff
@@ -70,72 +86,69 @@ class CardSearchQueryBuilder
-        'trash_ability' => 'cards.trash_ability',
-        'trash_cost' => 'cards.trash_cost',
-        'u' => 'cards.is_unique',
-        'universal_faction_cost' => 'unified_restrictions.universal_faction_cost',
-        'v' => 'cards.agenda_points',
-        'x' => 'cards.stripped_text',
+        '_' => 'unified_cards.stripped_title',
+        'additional_cost' => 'unified_cards.additional_cost',
+        'advanceable' => 'unified_cards.advanceable',
+        'advancement_cost' => 'unified_cards.advancement_requirement',
+        'agenda_points' => 'unified_cards.agenda_points',
+        'base_link' => 'unified_cards.base_link',
+        'card_cycle' => 'unified_cards.card_cycle_ids',
+        'card_pool' => 'unified_cards.card_pool_ids',
+        'card_set' => 'unified_cards.card_set_ids',
+        'card_subtype' => 'unified_cards.lower_card_subtype_names',
+        'card_subtype_id' => 'unified_cards.card_subtype_ids',
+        'card_type' => 'unified_cards.card_type_id',
+        'cost' => 'unified_cards.cost',
+        'd' => 'unified_cards.side_id',
+        'eternal_points' => 'unified_cards.restrictions_points',
+        'f' => 'unified_cards.faction_id',
+        'faction' => 'unified_cards.faction_id',
+        'format' => 'unified_cards.format_ids',
+        'g' => 'unified_cards.advancement_requirement',
+        'gains_subroutines' => 'unified_cards.gains_subroutines',
+        'h' => 'unified_cards.trash_cost',
+        'has_global_penalty' => 'unified_cards.restrictions_global_penalty',
+        'in_restriction' => 'unified_cards.in_restriction',
+        'influence_cost' => 'unified_cards.influence_cost',
+        'interrupt' => 'unified_cards.interrupt',
+        'is_banned' => 'unified_cards.restrictions_banned',
+        'is_restricted' => 'unified_cards.restrictions_restricted',
+        'is_unique' => 'unified_cards.is_unique',
+        'l' => 'unified_cards.base_link',
+        'link_provided' => 'unified_cards.link_provided',
+        'm' => 'unified_cards.memory_cost',
+        'memory_usage' => 'unified_cards.memory_cost',
+        'mu_provided' => 'unified_cards.mu_provided',
+        'n' => 'unified_cards.influence_cost',
+        'num_printed_subroutines' => 'unified_cards.num_printed_subroutines',
+        'num_printings' => 'unified_cards.num_printings',
+        'o' => 'unified_cards.cost',
+        'on_encounter_effect' => 'unified_cards.on_encounter_effect',
+        'p' => 'unified_cards.strength',
+        'performs_trace' => 'unified_cards.performs_trace',
+        'printing_id' => 'unified_cards.printing_ids',
+        'recurring_credits_provided' => 'unified_cards.recurring_credits_provided',
+        'restriction_id' => 'unified_cards.restriction_ids',
+        's' => 'unified_cards.lower_card_subtype_names',
+        'side' => 'unified_cards.side_id',
+        'snapshot' => 'unified_cards.snapshot_ids',
+        'strength' => 'unified_cards.strength',
+        't' => 'unified_cards.card_type_id',
```

Comment:
`type:...` and `subtype:...` are missing from the accepted syntax

Reply:
we have card_type and card_subtype already, actually.  I'd prefer to keep other syntax changes not related to the backing view change out of this PR, but let's revisit when i put the syntax doc page together.  sg?